catalyst-gateway/bin/src/service/api/legacy/mod.rs

```diff
@@ -0,0 +1,13 @@
+//! Legacy endpoints
```

Comment:
As far as I can see, this file was removed. Do we still need or is it a merge artifact?